pkg/booleanpolicy/field_metadata.go

```diff
@@ -396,7 +396,7 @@ func initializeFieldMetadata() FieldMetadata {
 		[]RuntimeFieldType{}, negationForbidden, operatorsForbidden)
 
 	f.registerFieldMetadata(fieldnames.ImageSignatureVerifiedBy,
-		querybuilders.ForFieldLabel(augmentedobjs.ImageSignatureVerifiedCustomTag),
+		querybuilders.ForImageSignatureVerificationStatus(),
```

Comment:
Since we removed negation, should we add `negationForbidden` to the field metadata like we do for i.e. `ImageScanAge`?

Reply:
That could be an option, if we don't want to allow users to be alerted on signatures from "untrusted" verifiers. Like keep the possibility to negate the negated policy. Looks like technically there's no need to disallow that.